Libraries/Text/Text/RCTTextShadowView.m

```diff
@@ -298,9 +301,9 @@ - (void)layoutSubviewsWithContext:(RCTLayoutContext)layoutContext
                         layoutDirection:self.layoutMetrics.layoutDirection
                           layoutContext:localLayoutContext];
 
-      // Reinforcing a proper frame origin for the Shadow View.
       RCTLayoutMetrics localLayoutMetrics = shadowView.layoutMetrics;
-      localLayoutMetrics.frame.origin = frame.origin;
+      localLayoutMetrics.frame.origin = frame.origin; // Reinforcing a proper frame origin for the Shadow View.
```

Comment:
Why not just set `display: none` here instead of introducing a new prop?
`localLayoutMetrics.display = RCTDisplayTypeNone;`

Reply:
@shergin good question. It's possible that my lack of familiarity with the code incorrectly turned me off of this approach.

I was afraid that setting `localLayoutMetrics.display = RCTDisplayTypeNone;` might prevent layout from running on subsequent layout passes and consequently preventing `localLayoutMetrics.display` from ever being updated back to the visible value (also, how do I make sure to change it back to the value the user set -- `RCTDisplayTypeFlex` vs `RCTDisplayTypeInline`?). I also had a concern about multiple pieces of code fighting to set the same field (what if between the time I set `display` and rendering somebody overwrites my value?).

Are my concerns unfounded? Is `shadowView.layoutMetrics` recreated from scratch on each layout pass? Please enlighten me 😄.

Reply:
I think this would be totally legit and work.
An `RCTLayoutMetrics` thing represents final computed layout metrics and can/should not be used as some input information for future layout passes; it's a product meant to be consumed by mounting layer.

If it's a View (and it it a View), it always should be `RCTDisplayTypeFlex`. Inline is reserved for VirtualText nodes.